### PR TITLE
Destripe preprocessor

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Cool.
One question : do we need a margin for that ?
Could you include in tests something that make the same computation with several chunk_size and verify that we have exactly the same results.

**Comment by alejoe91** [comment]
@samuelgarcia still a draft, maybe we can start collaborating with @oliche for this :)

**Comment by samuelgarcia** [comment]
I know I know but this is something I wanted to point out.
Lets wait olivier expertise.

**Comment by samuelgarcia** [comment]
@alejoe91  @oliche : I add the taper and also fix the odd/even rfft stuff.
Tell me what you think.

File: spikeinterface/toolkit/preprocessing/phase_shift.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            self.add_recording_segment(rec_segment)
+        
+        # for dumpability
+        if inter_sample_shift is not None:
+            inter_sample_shift = list(inter_sample_shift)
+        self._kwargs = dict(recording=recording.to_dict(), margin_ms=float(margin_ms),
+                             inter_sample_shift=inter_sample_shift)
+
+
+class DestripeRecordingSegment(BasePreprocessorSegment):
```
Comment (samuelgarcia) [inline spikeinterface/toolkit/preprocessing/phase_shift.py:58]: Phase shift !!!

**Pull request merged**